pkgs/development/python-modules/gidgethub/default.nix

```diff
@@ -16,6 +16,7 @@
 buildPythonPackage rec {
   pname = "gidgethub";
   version = "3.0.0";
+  format = "flit";
```

Comment:
the tarball comes with a setup.py, so that should work?

Reply:
I thought so too but it does not build without `format = "flit";`. Here is the error message.

```bash
these derivations will be built:
  /nix/store/qw39libx4yhj00c7wxz9wjnjvkykq343-python3.7-gidgethub-3.0.0.drv
building '/nix/store/qw39libx4yhj00c7wxz9wjnjvkykq343-python3.7-gidgethub-3.0.0.drv'...
unpacking sources
unpacking source archive /nix/store/zpnz9frn34alsi4a5nkd0f95vl17k6zb-gidgethub-3.0.0.tar.gz
source root is gidgethub-3.0.0
setting SOURCE_DATE_EPOCH to timestamp 1536623112 of file gidgethub-3.0.0/README.rst
patching sources
configuring
building
error in gidgethub setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
builder for '/nix/store/qw39libx4yhj00c7wxz9wjnjvkykq343-python3.7-gidgethub-3.0.0.drv' failed with exit code 1
```

Reply:
Should be ready to merge if this is fine.